bluebrain/repo-bluebrain/packages/libsonata/package.py

```diff
@@ -30,6 +30,8 @@ class Libsonata(CMakePackage):
     version("0.1.11", tag="v0.1.11")
     version("0.1.10", tag="v0.1.10")
 
+    patch("0001-Include-Catch2-only-if-building-tests.patch")
```

Comment:
Could you please add a comment here saying that this needs to be removed in the next version of `libsonata` and why this was introduced?